build.sbt

```diff
@@ -1,32 +1,41 @@
+import Dependencies.versions
+
+ThisBuild / version := "0.1.0-SNAPSHOT"
+ThisBuild / organization := "com.github.scalacn"
+ThisBuild / organizationName := "exchange"
+ThisBuild / scalaVersion := versions.scala2
```

Comment:
move these back to the root setting should be fine

Reply:
All the sub projects' scalaVersion will be 2.12 if changed.